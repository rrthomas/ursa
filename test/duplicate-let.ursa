let a = 1 and let a = 2
