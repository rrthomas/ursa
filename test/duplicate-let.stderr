Error: Line 1, col 1:
> 1 | let a = 1 and let a = 2
      ^~~~~~~~~~~~~~~~~~~~~~~
  2 | 

Duplicate identifier in let: a